erc190/package.py

```diff
@@ -29,6 +29,23 @@ def __init__(self, lockfile):
 
         try:
             validate(package_data, schema_data)
-        except jsonValidationError:
+        except:
             raise ValidationError
         self.package_data = package_data
+
+    def __repr__(self):
+        """
+        This can be expanded to include the
+        hashed content as discussed once
+        Package.hash exists
+        """
+
+        name = self.package_data['package_name']
+        version = self.package_data['version']
+        return "<Package " + name + "==" + version + ">"
```

Comment:
This is somewhat of a preference but, for consistency, can you change this to construct the string as follows.

```
return "<Package {0}=={1}>".format(name, version)
```